agasc/supplement/magnitudes/star_obs_catalogs.py

```diff
@@ -1,30 +1,48 @@
 import os
-import sys
 import numpy as np
 import tables
 
-from astropy.table import Table
+from astropy.table import Table, join
 from astropy import table
 
 from Ska.DBI import DBI
 from cxotime import CxoTime
 from Chandra.Time import date2secs
 from chandra_aca.transform import yagzag_to_pixels
 
-TIMELINES = None
-STARCAT_CMDS = None
-DWELLS_NP = None
+_STARCAT_CMDS = None
+_DWELLS = None
 """Array of dwells (mp_starcat_time, tstart, tstop, ra, dec, roll)."""
 
-DWELLS_MAP = None
-"""Time-index on DWELLS_NP. This is a dictionary mapping starcat times to an index in DWELLS_NP."""
-
 STARS_OBS = None
 """The table of star observations"""
 
-STARS_OBS_INDICES = None
-STARS_OBS_MAP = None
-STARS_OBS_NP = None
+
+def get_star_observations(start=None, stop=None, obsid=None, join_keys=['starcat_date', 'obsid']):
+    from kadi import commands
```

Comment:
This import doesn't need to be local.

Reply:
I moved it out of the function. This import indeed needed to be local when kadi was django based, because of the multi-process option.